### PR TITLE
Add rust formatting config

### PR DESCRIPTION
**Pull request by Kryptos-FR**
Format existing code using `cargo fmt` according to that config.

### CODE REVIEW

**Comment by sinkingsugar** [comment]
Nice,
How does that work? do we have to run manually `cargo fmt`? cos I generally do it via vscode already usually (manually)

Also btw, you should remove `rust/src/chainblocksc.rs`
That file needs some mention on the readme maybe. Basically ti should not be there but it's there just in order to let the Windows 32bit build pass, cos I push that specific version of the bindings, otherwise they cannot be generated in CI.

File: rust/rustfmt.toml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+newline_style = "Native"
```
Comment (sinkingsugar) [inline rust/rustfmt.toml:1]: what are the other styles?

**Review by sinkingsugar — CHANGES_REQUESTED** [review]

File: rust/rustfmt.toml (hunk 1 above)
Comment (Kryptos-FR) [reply rust/rustfmt.toml:1]: Possible values: "Auto", "Native", "Unix", "Windows"

Reference: https://rust-lang.github.io/rustfmt/?version=v1.4.36&search=#newline_style

**Comment by Kryptos-FR** [comment]
> Also btw, you should remove `rust/src/chainblocksc.rs`

Ok.

**Review by sinkingsugar — APPROVED** [review]

**Pull request merged**